zenodo/modules/records/serializers/schemas/json.py

```diff
@@ -248,6 +249,16 @@ class RecordSchemaV1(common.CommonRecordSchemaV1):
     revision = fields.Integer(dump_only=True)
     updated = fields.Str(dump_only=True)
 
+    _stats = fields.Method('dump_stats')
+
+    def dump_stats(self, obj):
+        """Dump the stats to a dictionary."""
+        if '_stats' in obj:
+            return obj['_stats']
+        else:
+            return build_stats(obj['metadata']['recid'],
```

Comment:
BTW a nice extra test worth adding is:

1. Mock `build_stats`
2. Perform a Search API request
3. Checking if the mock was called [via `.called`](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.called)